src/components/Chat/Chat.tsx

```diff
@@ -71,19 +58,156 @@ function anonymizeChatFiltersForPosthog(chatFilters: ChatFilters): AnonymizedCha
+  currentChatHistory,
+  isAddContextFiltersModalOpen,
+  chatFilters,
+  setChatFilters,
+  setUserTextFieldInput,
+  defaultModelName,
+  vaultDirectory,
+  setIsAddContextFiltersModalOpen,
+  handlePromptSelection,
+  askText,
+  loadAnimation,
+}) => {
+  const getClassName = (message: ChatMessageToDisplay): string => {
+    return message.messageType === 'error'
+      ? `markdown-content ${message.messageType}-chat-message text-white`
+      : `markdown-content ${message.role}-chat-message`
+  }
+
+  return (
+    <div
+      ref={chatContainerRef}
+      className="chat-container relative flex h-full flex-col items-center justify-center overflow-auto bg-transparent"
+    >
+      <div className="relative mt-4 flex size-full flex-col items-center gap-3 overflow-x-hidden p-10 pt-0">
+        <div className="w-full max-w-3xl flex-col items-center">
+          {currentChatHistory && currentChatHistory.displayableChatHistory.length > 0 ? (
+            // Display chat history if it exists
+            currentChatHistory.displayableChatHistory
+              .filter((msg) => msg.role !== 'system')
+              .map((message, index) => (
+                <div className={`w-full ${getClassName(message)} flex`}>
+                  <div className="relative items-start pl-4 pt-3">
+                    {message.role === 'user' ? (
+                      <FaRegUserCircle size={22} />
+                    ) : (
+                      <img src="/src/assets/reor-logo.svg" style={{ width: '22px', height: '22px' }} alt="ReorImage" />
+                    )}
+                  </div>
+                  <div className="w-full flex-col gap-1">
+                    <div className="flex grow flex-col px-5 py-2.5">
+                      <ReactMarkdown
+                        // eslint-disable-next-line react/no-array-index-key
+                        key={index}
+                        rehypePlugins={[rehypeRaw]}
+                        className="max-w-[95%] break-words"
+                      >
+                        {message.visibleContent
+                          ? message.visibleContent
+                          : formatOpenAIMessageContentIntoString(message.content)}
+                      </ReactMarkdown>
+                    </div>
+                  </div>
+                </div>
+              ))
+          ) : (
+            // Display centered "Start a conversation..." if there is no currentChatHistory
+            <div className="relative flex w-full flex-col">
+              <div className="relative flex size-full flex-col text-center lg:top-10">
+                <div className="flex size-full justify-center">
+                  <img src="/src/assets/reor-logo.svg" style={{ width: '64px', height: '64px' }} alt="ReorImage" />
+                </div>
+                <h1 className="mb-10 text-[28px] text-gray-300">
+                  Welcome to your AI-powered assistant! Start your first conversation or pick up where you left off.
```

Comment:
I think the second sentence after exclamation should be:

Start a conversation with your second brain...